### PR TITLE
Add auth bulk get/delete snippets

### PR DESCRIPTION
**Pull request by samtstern**
Based on:
https://github.com/firebase/snippets-node/pull/86

### CODE REVIEW

File: admin/src/main/java/com/google/firebase/example/FirebaseAuthSnippets.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    // [START bulk_get_users]
+    GetUsersResult result = FirebaseAuth.getInstance().getUsersAsync(Arrays.asList(
+        new UidIdentifier("uid1"),
+        new EmailIdentifier("user2@example.com"),
+        new PhoneIdentifier("+15555550003"),
+        new ProviderIdentifier("google.com", "google_uid4"))).get();
+
+    System.out.println("Successfully fetched user data:");
+    for (UserRecord user : result.getUsers()) {
+      System.out.println(user);
```
Comment (rsgowman) [inline admin/src/main/java/com/google/firebase/example/FirebaseAuthSnippets.java]: UserRecord doesn't have a (useful) toString method. Maybe that doesn't matter.

As an alternative, you could create some sort of helper method that would print out something a little more useful... though that might clutter the snippet somewhat. I'm not sure I'd recommend that.

Another alternative would be to print out the uid of the user, i.e. `System.out.println(user.getUid());`

Optional.

File: admin/src/main/java/com/google/firebase/example/FirebaseAuthSnippets.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    System.out.println("Unable to find users corresponding to these identifiers:");
+    for (UserIdentifier uid : result.getNotFound()) {
+      System.out.println(uid);
```
Comment (rsgowman) [inline admin/src/main/java/com/google/firebase/example/FirebaseAuthSnippets.java:47]: Unlike the above, UserIdentifier *does* have a (useful) toString method, so this is fine. No action required.

File: admin/src/main/java/com/google/firebase/example/FirebaseAuthSnippets.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  public static void bulkDeleteUsers() throws Exception {
+    // [START bulk_delete_users]
+    DeleteUsersResult result = FirebaseAuth.getInstance().deleteUsersAsync(
+        Arrays.asList("uid1", "uid2", "uid3")).get();
+
+    System.out.println("Successfully deleted " + result.getSuccessCount() + " users");
+    System.out.println("Failed to delete " + result.getFailureCount() + " users");
+    for (ErrorInfo error : result.getErrors()) {
+      System.out.println(error);
```
Comment (rsgowman) [inline admin/src/main/java/com/google/firebase/example/FirebaseAuthSnippets.java]: No useful toString on ErrorInfo either. How about:

System.out.println("Index: " + error.getIndex() + "; Reason: " + error.getReason());

**Review by rsgowman — APPROVED** [review]

File: admin/src/main/java/com/google/firebase/example/FirebaseAuthSnippets.java (hunk 1 above)
Comment (samtstern) [reply admin/src/main/java/com/google/firebase/example/FirebaseAuthSnippets.java]: Thanks!

**Pull request merged**